extensions/project-descriptor.md

```diff
@@ -186,6 +198,10 @@ name = "JAVA_OPTS"
 value = "-Xmx1g"
 ```
 
+## Definitions
+
+* **Inline Buildpack** - a type of buildpack that can be defined in the same repo as the app it is used with
```

Comment:
For [consistency](https://github.com/buildpacks/spec/blob/main/platform.md#terminology) maybe we should call this terminology?

Reply:
fixed in ea91afc